lib/matplotlib/pyplot.py

```diff
@@ -728,6 +728,11 @@ def figure(num=None,  # autoincrement if None, else integer from 1-N
     `~matplotlib.rcParams` defines the default values, which can be modified
     in the matplotlibrc file.
     """
+    if isinstance(num, Figure):
+        if num.canvas.manager is None:
+            raise ValueError("Figure is not managed by pyplot")
```

Comment:
Can we make that clearer/give it context? Not sure how. Maybe:
```suggestion
            raise ValueError("The passed figure is not managed by pyplot")
```

Reply:
sure